prime-angular-frontend/src/app/modules/auth/auth-routing.module.ts

```diff
@@ -4,7 +4,10 @@ import { Routes, RouterModule } from '@angular/router';
 import { AuthComponent } from './shared/components/auth/auth.component';
 import { AuthorizationRedirectGuard } from './shared/guards/authorization-redirect.guard';
 
+import { AuthRoutes } from './auth.routes';
+
 import { InfoComponent } from './pages/info/info.component';
+import { AdminComponent } from './pages/admin/admin.component';
 
 const routes: Routes = [
   {
```

Comment:
The base route for auth can be replaced with the `MODULE_PATH` route constant.

Reply:
okay!